__init__.py

```diff
@@ -13,6 +13,13 @@
 
 class ConnectorGitHub(Connector):
 
+    @staticmethod
+    def _getSite(url):
```

Comment:
Please switch this method to the [PEP8 underscore naming convention](https://www.python.org/dev/peps/pep-0008/#function-names)